sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetRelation.scala

```diff
@@ -314,8 +312,6 @@ private[sql] class DefaultSource extends FileFormat with DataSourceRegister with
         inputFormatClass = classOf[ParquetInputFormat[InternalRow]],
         valueClass = classOf[InternalRow]) {
 
-        val cacheMetadata = useMetadataCache
```

Comment:
Oh, sorry. I missed this line. I did not realize that this conf is actually used at here...